### PR TITLE
Patch/add locale strings

### PR DESCRIPTION
**Pull request by Tama63**

### CODE REVIEW

File: applications/dashboard/views/profile/edit.php [hunk 1]
```diff
@@ -8,7 +8,7 @@
 <ul>
    <li class="User-Name">
       <?php
-         echo $this->Form->Label('Username', 'Name');
+         echo $this->Form->Label(T('Username'), 'Name');
```
Comment (tburry) [inline applications/dashboard/views/profile/edit.php]: `Gdn_Form->Label()` already passes the argument through `T()` so this is redundant.

File: applications/dashboard/views/search/index.php [hunk 2]
```diff
@@ -6,7 +6,7 @@
 echo  $Form->Open(array('action' => Url('/search'), 'method' => 'get')),
    '<div class="SiteSearch InputAndButton">',
    $Form->TextBox('Search'),
-   $Form->Button('Search', array('Name' => '')),
+   $Form->Button(T('Search'), array('Name' => '')),
```
Comment (tburry) [inline applications/dashboard/views/search/index.php]: `Gdn_Form->Button()` passes the argument through `T()` so this is redundant.

File: applications/dashboard/views/settings/gettingstarted.php [hunk 3]
```diff
@@ -172,7 +172,7 @@ function TutLink($TutorialCode, $WriteTitle = TRUE, $ThumbnailSize = 'medium') {
       <div class="Video"><?php echo TutLink('introduction', FALSE, 'large'); ?></div>
    </div>
    <div class="Step">
-      <div class="NumberPoint">1</div>
+      <div class="NumberPoint"><?php echo T('1'); ?></div>
```
Comment (tburry) [inline applications/dashboard/views/settings/gettingstarted.php]: Translating numbers as they are here is inappropriate.

File: plugins/AllViewed/class.allviewed.plugin.php [hunk 4]
```diff
@@ -25,8 +25,8 @@
  */
 
 $PluginInfo['AllViewed'] = array(
-   'Name' => 'All Viewed',
-   'Description' => 'Allows users to mark all discussions as viewed and mark category viewed.',
+   'Name' => T('All Viewed'),
```
Comment (tburry) [inline plugins/AllViewed/class.allviewed.plugin.php]: Info arrays must be static content and cannot take a php function.

**Comment by tburry** [comment]
I've added some comments on individual lines, but in general there are a lot of translations in this PR that are redundant or inappropriate. There are some good translations in there, but until the others are removed I can't merge this PR.

**Comment by Tama63** [comment]
Okay,

Thanks for the heads up, I was looking at the open issues on Github for
this PR - may I suggest closing the issues referenced in the unsuitable
commits so this type of issue doesn't crop up again.

Will look into redoing the PR when I get back to my machine this Sunday.

Cheers for the feedback

**Comment by Tama63** [comment]
Closing pending a review,

**Pull request closed without merging**